docs/src/documents.md

```diff
@@ -176,25 +176,22 @@ package:
 * _Pronouns_ : "I", "you", "he", "she", ...
 * _Stop Words_ : "all", "almost", "alone", ...
 
-These special classes can all be removed using specially-named functions:
+These special classes can all be removed using specially-named parameters:
 
-* `remove_articles!()`
-* `remove_indefinite_articles!()`
-* `remove_definite_articles!()`
-* `remove_prepositions!()`
-* `remove_pronouns!()`
-* `remove_stop_words!()`
+* `prepare!(sd, strip_articles)`
+* `prepare!(sd, strip_indefinite_articles)`
+* `prepare!(sd, strip_definite_articles)`
+* `prepare!(sd, strip_preposition)`
+* `prepare!(sd, strip_pronouns)`
+* `prepare!(sd, strip_stopwords)`
+* `prepare!(sd, strip_numbers)`
+* `prepare!(sd, strip_non_letters)`
+* `prepare!(sd, strip_spares_terms)`
+* `prepare!(sd, strip_frequent_terms)`
+* `prepare!(sd, strip_html_tags)`
```

Comment:
Here, add something like "These operations may be combined for performance: `prepare!(sd, strip_articles | strip_numbers | strip_html_tags)`"